VMEncryption/test/test_common.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import os
+import main.Common
+import json
+
+class TestCommonVariables(unittest.TestCase):
+    
+    def test_common_variables(self):
+        self.assertEqual(main.Common.CommonVariables.extension_name, "AzureDiskEncryptionForLinux")
+        self.assertEqual(main.Common.CommonVariables.extension_version, "1.1.0.47")
```

Comment:
Instead of hardcoding values, it would be good if values can be directly read from file using File I/O and compared to values in Common.py. Otherwise we will need to manually upgrade the tests on every version upgrade. I am fine if you want to this change in another PR.

Reply:
Good catch - thank you! I forgot to remove those hardcoded values. PR updated.